tools/profiling/memory/memory_diff.py

```diff
@@ -81,24 +91,27 @@ def _run():
     # checkout the diff base (="old")
     subprocess.check_call(['git', 'checkout', args.diff_base])
     try:
-        new = _run()
+        old = _run()
     finally:
-        # restore the original revision (="new")
+        # restore the original revision (="cur")
         subprocess.check_call(['git', 'checkout', where_am_i])
 
 text = ''
-if new is None:
-    for key, value in cur.items():
+if old is None:
+    for key, value in sorted(cur.items()):
         text += '{}: {}\n'.format(key, value)
 else:
+    print(cur, old)
```

Comment:
`cur` is printed above. Is this redundant?

Reply:
it is a bit I guess, might remove the first